packages/reports/src/reports/blocks/types/index.js

```diff
@@ -0,0 +1,85 @@
+import {BLOCK_TYPES} from "../../../utils/consts/cms";
+
+import adapters from "./adapters";
+import editors from "./simpleEditors";
+import renderers from "./renderers";
```

Comment:
Just want to say that this little three-spot of imports makes me happy. I'm glad we talked this out on the whiteboard